lib/soak/configs/observer.yaml

```diff
@@ -1,9 +1,12 @@
-prometheus: "http://localhost:9090"
 experiment_name: "foobar"
 variant: "baseline"
-vector_id: "featuresha-commitsha"
-queries:
-  - query: sum(rate((bytes_written[2s])))
-    id: throughput
-    unit: bytes
 capture_path: "/tmp/captures-baseline.txt"
+targets:
+  - id: vector
+    url: "http://localhost:9598/metrics"
+  - id: lading_http_gen
+    url: "http://localhost:9001"
+  - id: lading_http_blackhole
+    url: "http://localhost:9002"
+  - id: lading_tcp_gen
+    url: "http://localhost:9003"
```

Comment:
Oh yeah, now that you mention it. I hadn't considered this but I had been thinking that lading might benefit from a busybox kind of model.

Reply:
It may get tricky ensuring the both ends get the right ports assigned, but just a thought as I imagine this list growing with `900N` as ports 😆

Reply:
Yeah, it's annoying to keep this updated for sure, which is why I was leaning toward the busybox model. That would also make deployment more straightforward: now instead of multiple terraform modules to manage you just have the one and you feed it a single config file.